docs/configuration.md

```diff
@@ -1638,7 +1638,7 @@ Apart from these, the following properties are also available, and may be useful
     For more detail, see the description
     <a href="job-scheduling.html#dynamic-resource-allocation">here</a>.
     <br><br>
-    This requires <code>spark.shuffle.service.enabled</code> to be set.
+    This requires <code>spark.shuffle.service.enabled</code> to be set true.
```

Comment:
Since other places are clearly defined the property, so there should be no ambiguity. Personally I'm not fond of this super nit fix...